azure-mgmt-eventhub/src/main/java/com/microsoft/azure/management/eventhub/implementation/EventHubManager.java

```diff
@@ -0,0 +1,87 @@
+/**
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license
+information.
+Code generated by Microsoft (R) AutoRest Code Generator.
+ */
+
+package com.microsoft.azure.management.eventhub.implementation;
+
+import com.microsoft.azure.AzureEnvironment;
+import com.microsoft.azure.AzureResponseBuilder;
+import com.microsoft.azure.credentials.AzureTokenCredentials;
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.resources.fluentcore.arm.AzureConfigurable;
+import com.microsoft.azure.management.resources.fluentcore.arm.implementation.AzureConfigurableImpl;
+import com.microsoft.azure.management.resources.fluentcore.arm.implementation.Manager;
+import com.microsoft.azure.management.resources.fluentcore.utils.ProviderRegistrationInterceptor;
+import com.microsoft.azure.serializer.AzureJacksonAdapter;
+import com.microsoft.rest.RestClient;
+
+/**
+ * Entry point to Azure EventHub resource management.
+ */
+@Beta(Beta.SinceVersion.V1_2_0)
```

Comment:
We use `@Beta(SinceVersion.V1_2_0)` - I think you can configure this in IntelliJ settings -> editor -> code style -> Java -> Imports -> Insert imports for inner classes